Explorer/Assets/DCL/AuthenticationScreenFlow/AuthenticationScreenController.cs

```diff
@@ -105,16 +117,17 @@ protected override void OnBeforeViewShow()
         {
             base.OnBeforeViewShow();
 
+            viewInstance.gameObject.transform.SetSiblingIndex(2);
```

Comment:
Setting sibling index 2 seems too random for one who does not know how the view was made, can you add a comment? Since this is a controller we shouldn't even have to have access to the view's game object

Reply:
Even extracting the magic number into a named const could help